n2v/internals/N2V_DataGenerator.py

```diff
@@ -10,7 +10,7 @@ class N2V_DataGenerator():
     The 'N2V_DataGenerator' enables training and validation data generation for Noise2Void.
     """
 
-    def load_imgs(self, files, dims='YX'):
+    def load_imgs(self, files, to32bit, dims='YX'):
```

Comment:
This parameter should have a default value of `False`. This will ensure backwards compatibility.